mantidproject/mantid
# Added Crystal Field Examples to Techniques Folder In Docs

**Description of work.**
Added Crystal Field Examples to docs in techniques folder. Reworked from [https://www.mantidproject.org/Crystal_Field_Examples](https://www.mantidproject.org/Crystal_Field_Examples). Updated examples to now work. There are two headings commented out which are to be added at a later date.

**To test:**
Build docs and check Crystal Field Examples looks okay.

*There is no associated issue.*

*This does not require release notes* because **it is a small addition to the docs**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
